app/controllers/admin/instances_controller.rb

```diff
@@ -1,5 +1,97 @@
 class Admin::InstancesController < AdminController
```

Comment:
Is it correct that there is no "hand" when you hover on the dropdown ? You can't know it's clickable

![image](https://user-images.githubusercontent.com/8007722/73747689-f51e7f00-4725-11ea-8136-a6a5f1aa25d6.png)